recipes/r5py.sampledata.sao_paulo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "r5py.sampledata.sao_paulo" %}
+{% set version = "0.1.1.dev0" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
We now updated it to stable version 0.1.2